packages/app-store/jelly/api/callback.ts

```diff
@@ -0,0 +1,72 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import prisma from "@calcom/prisma";
+
+import getAppKeysFromSlug from "../../_utils/getAppKeysFromSlug";
+import getInstalledAppPath from "../../_utils/getInstalledAppPath";
+import createOAuthAppCredential from "../../_utils/oauth/createOAuthAppCredential";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { code } = req.query;
+  const { client_id, client_secret } = await getAppKeysFromSlug("jelly");
+
+  const result = await fetch(`https://jellyjelly.com/login/oauth/access_token`, {
+    method: "POST",
```

Comment:
For some reason I was getting a `TypeError: fetch failed` error. Didn't happen when I switched to `axios`

Reply:
I believe this is fixed now!